### PR TITLE
Add an option flag to not display icons but text instead

### PR DESCRIPTION
**Pull request by phacops**
I didn't want to set up the font (sorry :D) so I translated everything as text. It's now just an option when you start the daemon (`-text`).

Also, I had a strange device name for my wifi card so I added an option to specify a different dev name (`-dev`).

To finish, instead of an empty string at the beginning, I'm displaying the machine's hostname.

Comments are more than welcome.

### CODE REVIEW

File: gods.go [hunk 1]
```diff
@@ -26,6 +27,26 @@ var (
 	cores = runtime.NumCPU() // count of cores to scale cpu usage
 	rxOld = 0
 	txOld = 0
+
+	textOnly   = flag.Bool("text", false, "no color mode")
```
Comment (schachmat) [inline gods.go]: this should be called "no icon mode" since it does not affect the color codes.

**Comment by schachmat** [comment]
Thanks for your contribution. Since this repository is mainly meant as an example to people, I won't merge the pull-request. The users are expected to adapt it to their needs, as you have done (I will add a note about this to the readme). If I would add flags for every possibility a user could want to have his status line look like, this would turn out to be a huge mess.

If you want, you can supply a pull-request, which only introduces the `…Sign` and `…Separator` variables (no flags), so it can be easily adapted to peoples needs. This one I would merge.

Also I would link to your fork in the readme as another example for people to look at, if this is ok for you?

**Comment by phacops** [comment]
Seems very reasonable. I'll delete the flag for the text mode and transform everything in constants instead of vars then.

What about the dev flag? Are you OK with this one?

**Comment by schachmat** [comment]
I think it would be more consistent to have all configuration possibilities as variables than mixing variables with flags, so you can just introduce another global variable for this as well. Maybe add a comment above the variables explaining that they are meant for configuration.

**Comment by phacops** [comment]
The comment is not needed, you already explain to change the values if we don't use the font at the top of the file.

**Comment by phacops** [comment]
And yes, you can link my fork in the README for another example to look at.

**Comment by schachmat** [comment]
Ok, linked you in the comments. Please remove the `gods` binary from your pull request and squash it to a single commit with `git rebase -i`.

**Comment by phacops** [comment]
I think we're all set now. Thanks for the review.

**Comment by schachmat** [comment]
I removed the blank lines and the getHostname function before merging it. So you will have to add the hostname function for your own repository again on top of the new master. I also added a new commit, simplifying customization of all network interfaces to be used.
Thanks again for your contribution.

**Comment by phacops** [comment]
Thank you for the merge.

**Pull request closed without merging**